src/Preflight/Preflight.php

```diff
@@ -263,7 +263,9 @@ public function preflight($argv)
 
         // Process the selected alias. This might change the selected site,
         // so we will add new site-wide config location for the new root.
-        $root = $this->setSelectedSite($selfAliasRecord->localRoot());
+        if ($localRoot = $selfAliasRecord->localRoot()) {
```

Comment:
Minor style point: I'd rather see this `if` inside of the `setSelectedSite()` method rather than here.